docs/pages/machine-id/workload-identity/aws-roles-anywhere.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Configuring Workload ID and AWS Roles Anywhere
+description: Configuring AWS to accept Workload ID certificates as authentication using AWS Roles Anywhere
+---
+
+Teleport's Workload Identity issues flexible short-lived identities in X.509
```

Comment:
```suggestion
Teleport Workload Identity issues flexible short-lived identities in X.509
```

Since this doesn't belong to Teleport (i.e., as a possessive) but is a Teleport feature.